.github/workflows/release.yaml

```
@@ -45,12 +44,43 @@ jobs:
       - name: creds
         run: gcloud auth configure-docker --quiet
 
-      - name: release
-        run: make release
+      - name: Set LDFLAGS
+        id: ldflags
+        run: |
+           source ./release/ldflags.sh
+           goflags=$(ldflags)
+           echo "goflags="${goflags}"" >> "$GITHUB_OUTPUT"
```

since you will use the value in the same job replace the output for `GITHUB_ENV`

and then you can use `env.GO_FLAGS`


```suggestion
           echo "GO_FLAGS="${goflags}"" >> "$GITHUB_ENV"
```